### ISSUE TITLE
Horribly Bad Accuracy

### ISSUE THREAD

**Issue by Syzygianinfern0** [sachin-101/Machine-Learning-by-Andrew-Ng-Implementation-in-Python#1]
A normalization on the data set gives an accuracy of about 85% on just 5 epochs. If still no, some stupid error like your '+' is hiding 😁 

This is wrt [The neural network here](https://github.com/Sachinkumar04428/Machine-Learning-by-Andrew-Ng-Implementation-in-Python/blob/master/Neural%20Networks%20from%20Scratch/Multiclass%20Classification%20Neural%20Network/Multiclassification_Neural_Network_from_Scratch_1.ipynb)

**Comment by sachin-101** [comment]
Thanks bro, for pointing out the low accuracy.
I will correct it soon.

**Comment by sachin-101** [comment]
@Syzygianinfern0  Did you do the normalization of the data and checked it yourself ? Or are you predicting that very few epochs would be enough.

**Comment by sachin-101** [comment]
@Syzygianinfern0  After Normalizing the data, even after 100 epochs the accuracy is just 12.34%. 
5 epochs is too less for a Multi classification neural network to learn.

**Comment by sachin-101** [comment]
As I had mentioned it in the notebook, the low accuracy was only due to less number of neurons used. I increased the number of neurons and trained the model to get an accuracy of 87%. Here is the [LINK](https://github.com/sachin-101/Machine-Learning-by-Andrew-Ng-Implementation-in-Python/blob/master/Neural%20Networks%20from%20Scratch/Neural%20net%20from%20Scratch%202%20_more_neurons.ipynb)...
 By training it a bit more, we can cross the 90 mark also.

**Comment by Syzygianinfern0** [comment]
> As I had mentioned it in the notebook, the low accuracy was only due to less number of neurons used. I increased the number of neurons and trained the model to get an accuracy of 87%. By training it a bit more, we can cross the 90 mark also.

Try adding CNNs. You can achieve almost perfect confidence at classification.
